ideals.md

```diff
@@ -10,75 +10,52 @@ These ideals also help define the scope of Cockpit. That scope is limited, and n
-
-### Cockpit does not store data or policy
-
-As a general rule, Cockpit does not have a data store other than the server configuration itself. A good example of this that we don’t have our own concept of users. We allow the server’s system users to authenticate using their system credentials. The same users that authenticate over SSH.
+### Cockpit requires no configuration or infrastructure
 
-Cockpit does not store its own opinion about the state of the server. The Cockpit UI reflects the actual current state of the server.
+The goal is that Cockpit comes with the default server install of a Linux server, except for really minimal installs. Cockpit comes “out of the box” ready for the admin to interact with the system immediately.
 
-Cockpit does not make security decisions. Once the user is authenticated they have exactly the same permissions as the user would have if they logged into the server via the console or SSH. When escalating the user’s permissions to perform a privileged action, we use the exact same mechanisms (eg: polkit or sudo) that a user would use at the command line.
+In cases where there is other infrastructure available we should allow the admin to configure the server to take advantage of it (e.g.: use single-sign-on against a domain controller). But such infrastructure is not a prerequisite.
 
 ### Cockpit is not configuration management
 
 Cockpit itself does not have a predefined template or state for the server that it then imposes on the server. It is imperative configuration rather than declarative configuration.
 
 Cockpit may expose UI elements that interact with configuration management style software installed on the server. Such software may have a declarative configuration, as one would expect. But Cockpit itself has no concept of configuration management, or a desired state for the server.
 
-### Cockpit reacts to the server
-
-Cockpit dynamically updates itself to reflect the current state of the server, within a time frame of a few seconds. This is important to the admin experience. An admin should feel that they are on the server itself, and not in some tool. The admin should be able to effectively use a terminal, Cockpit, and other management tools all concurrently.
+### Cockpit is only the project name
 
-### Cockpit is firewall friendly
+People using Cockpit should feel they are interacting with the underlying Server OS. Cockpit can look different on different operating systems, because it’s the UI for the OS, and not a external tool.
 
-Cockpit opens one port for browser connections: by default that is 9090. We work well in cases where a bastion host runs Cockpit and connects out to the other servers via SSH.
+When packaging a stable release of Cockpit for release with a given OS, most of the “Cockpit” branding should be removed, and OS specific branding is put in its place.
```

Comment:
Might worth mentioning that we minimize the use of the term "cockpit" in the UI as much as possible, that it's referred to as "Web Console" (or whatever term we use, I think it's slightly different in different places :/ ), and that a lot of the branding is taken care of automatically, such as parsing /etc/os-release for the product name, loading the OS logo from /usr/share/logos etc.

Reply:
Also, this isn't actually what happens - we don't "remove" branding during  packaging, we never have it in the first place. I would rather say this directly, like "... not an external tool. Thus the UI should never actually show the 'Cockpit' name, but call the server by its name or operating system." (The exception is the "About.." menu entry, not sure if we really need to spell that out - I think we shouldn't in the interest of brevity.)

Reply:
Thanks, good point! I didn't catch that (the diff shows this as new, but I've only moved it around). I've used @martinpitt's suggested sentence here. I think saying "but call the server by its name or operating system" implies that we get this information from the server. I also don't want to get too technical and detailed.